### DISCUSSION TITLE
No need to show "edit dashboard" button for yaml-driven dashboards

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#19494] [Dashboard] [answered]
The "Edit" button should not be displayed for yaml-driven dashboards:

![298346410-a3d562a7-cc4a-4d9e-9b3e-6f8510eacf1c](https://github.com/home-assistant/frontend/assets/71872483/b50d809b-4c42-4205-8138-f0a0fba339b8)

since it just causes this msgbox:

![298346354-5e7fa115-46cf-4a27-a61d-ad36f082ee09](https://github.com/home-assistant/frontend/assets/71872483/2cbcd2d2-8178-49b3-85e4-890585bcf93d)

**Comment by matthiasdebaat** [comment, marked as answer]
Though I understand you want to hide this. We need the explanation dialog for users who are not familiar with this.

**Reply by ildar170975** [reply]
Make sense.
